presto-resource-group-managers/src/main/java/com/facebook/presto/resourceGroups/db/ResourceGroupsDao.java

```diff
@@ -74,4 +74,19 @@
             "  FOREIGN KEY (resource_group_id) REFERENCES resource_groups (resource_group_id)\n" +
             ")")
     void createSelectorsTable();
+
+    @SqlUpdate("CREATE TABLE IF NOT EXISTS exact_match_selectors(\n" +
+            "  environment VARCHAR(128) NOT NULL,\n" +
+            "  query_identifier VARCHAR(512) NOT NULL,\n" +
+            "  update_time DATETIME NOT NULL,\n" +
+            "  resource_group_id VARCHAR(256) NOT NULL,\n" +
+            "  PRIMARY KEY(environment, query_identifier) \n" +
+            ")")
+    void createExactMatchSelectorsTable();
+
+    @SqlQuery("SELECT resource_group_id\n" +
+            "FROM exact_match_selectors\n" +
+            "WHERE environment = :environment \n" +
```

Comment:
There is space here before `\n`, but not in other places, I don't know the convention we have, but please make it consistent. Similarly, we don't have space before "FROM" and "WHERE" in this query, but above we have space before the first character in every new line. Anyway, this is just a tiny issue, but still good to be consistent.

Reply:
Good catch.